ote/src/cljs/ote/ui/select_field.cljs

```diff
@@ -33,9 +37,15 @@
          (map-indexed
            (fn [i option]
              (if (= :divider option)
-               ^{:key i}
+               ^{:key (str "select-field-divider")}
```

Comment:
If map-indexed has more than one element, key is not unique?

Reply:
True, `i` was supposed to be appended. Will fix, thanks.

Reply:
Although above applies if there are several `:divider`s